filebeat/beat/filebeat.go

```diff
@@ -101,17 +112,18 @@ func (fb *Filebeat) Cleanup(b *beat.Beat) error {
 // Stop is called on exit for cleanup
 func (fb *Filebeat) Stop() {
 
-	// Stop harvesters
-	// Stop prospectors
+	logp.Info("Stopping filebeat")
+	// Stop crawler -> stop prospectors -> stop harvesters
+	fb.cralwer.Stop()
```

Comment:
what happens if harvester is blocked by channel forwarding events to the spooler and spooler itself is blocked by libbeat due to ES/LS not responding?